adobe/cmath.hpp

```diff
@@ -5,170 +5,33 @@
 */
 /*************************************************************************************************/
 
-/*
-
-REVISIT (sparent) : Need to replicate the boost configuration tests to figure out when to fall
-back to include math.h. This also needs to add any other C99 math.h extensions.
-
-*/
-
 #ifndef ADOBE_CMATH_HPP
 #define ADOBE_CMATH_HPP
 
 #include <adobe/config.hpp>
 
+#include <cmath>
 #include <functional>
 
 /*************************************************************************************************/
 
-#if defined(__MWERKS__)
-/*
-    Any (previously) supported version of metrowerks had the C99/TR1 cmath extensions in the
-    standard namespace in <cmath>.
-*/
 #define ADOBE_HAS_C99_STD_MATH_H
```

Comment:
I confirm that VS2015u2 has the functions required in its std. And cmath test run fine, by the way.